src/sdk/pynni/nni/smac_tuner/smac_tuner.py

```diff
@@ -258,4 +262,45 @@ def generate_multiple_parameters(self, parameter_id_list):
         return params
 
     def import_data(self, data):
-        pass
+        """Import additional data for tuning
+        Parameters
+        ----------
+        data:
+            a list of dictionarys, each of which has at least two keys, 'parameter' and 'value'
+        """
+        _completed_num = 0
+        for trial_info in data:
+            self.logger.info("Importing data, current processing progress %s / %s", _completed_num, len(data))
+            # simply validate data format
+            assert "parameter" in trial_info
+            _params = trial_info["parameter"]
+            assert "value" in trial_info
+            _value = trial_info['value']
+            if not _value:
+                self.logger.info("Useless trial data, value is %s, skip this trial data.", _value)
+                continue
```

Comment:
you could try to avoid "continue" here...

Reply:
why?